cmd/config/aws/create_role_cmd.go

```diff
@@ -11,14 +11,28 @@ import (
 	"github.com/spf13/cobra"
 	log "go.uber.org/zap"
 	"io"
+	"time"
+)
+
+var (
+	useDevTemplate bool
 )
 
 const (
-	configCreateRoleShort = "AWS Create Role via CloudFormation Stack"
-	//TODO For usage documentation, visit...
-	configCreateRoleLong    = "Use AWS CloudFormation quick create to run a template that allows Armory to assume a role to manage deployments\n"
+	configCreateRoleShort   = "AWS Create Role via CloudFormation Stack"
+	configCreateRoleLong    = "Use Armory's AWS CloudFormation template to create a Stack with an IAM role. This IAM Role allows Armory to manage your Lambda deployments.\n"
 	configCreateRoleExample = "armory config aws create-role"
-	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://armory-cdaas-cloudformation.s3.us-west-2.amazonaws.com/iam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack&param_AccountId=%s&param_ExternalId=%s"
+	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://s3.us-west-2.amazonaws.com/static.cloud.armory.io/templates/aws/cloudformation/%siam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack-%s&param_AccountId=%s&param_ExternalId=%s&param_ArmoryRoleName=%s"
+	devTemplatePrefix       = "dev-"
+	armoryRoleName          = "ArmoryRole"
+
+	installPromptIntro  = "In order to deploy AWS resources, Armory needs to create a Trust Relationship in your AWS account by adding an IAM role that Armory can assume to execute deployments on your behalf."
+	installPromptPrereq = "Prerequisite: In your default browser, log in to the AWS Account you want to connect to Armory CD-as-a-Service. You must have permission to configure IAM roles."
+	installPromptStep1  = "1. Type \"Y\" to begin Stack creation. This opens your browser to the CloudFormation page of your AWS Console. You complete the rest of this process in your browser."
+	installPromptStep2  = "2. Review the resources that Armory is creating in your AWS account. The default IAM Role name is \"ArmoryRole\" but you can customize the name of that role if you wish."
+	installPromptStep3  = "3. Click \"Create\" on the AWS CloudFormation page and wait for Stack creation to complete."
+	installPromptStep4  = "4. After the CloudFormation Stack creation finishes, locate the created role ARN in the \"Outputs\" section. You can find it under the key \"RoleArnOutput\". You use the ARN to populate the \"account\" field in your CD-as-a-Service Lambda deployment target."
```

Comment:
```suggestion
	installPromptStep4  = "4. After the CloudFormation Stack creation finishes, locate the created role ARN in the \"Outputs\" section. You can find it under the key \"RoleArnOutput\". You use the ARN to populate the \"deployAsRoleArn\" field in your CD-as-a-Service Lambda deployment target."
```

https://github.com/armory-io/infra-armory-cloud-api-tests/blob/main/resources/lambda/minimal.yaml#L8